server/dao/games.js

```diff
@@ -13,11 +15,25 @@
 
 /**
  * get game
- * @param {string} id - game id
+ * @param {string} gameId - game id
  * @returns {object} - object containing user ids and game state
  */
-function getGame(id) {
-  // TODO:
+function getGame(gameId, token) {
+  // TODO: remove auth for this
+  const url = `http://localhost:8080/state/${gameId}/currentState`;
```

Comment:
this wont work in docker 
(localhost)

Reply:
I will add those strings in env file, for now this is fine